telnyx_rtc/src/main/java/com/telnyx/webrtc/sdk/TelnyxClient.kt

```diff
@@ -214,14 +214,19 @@ class TelnyxClient(
             val inviteCallId: UUID = UUID.randomUUID()
 
             callId = inviteCallId
+            val call = this
 
             // Create new peer
             peerConnection = Peer(
                 context, client, providedTurn, providedStun, callId.toString(),
                 object : PeerConnectionObserver() {
                     override fun onIceCandidate(p0: IceCandidate?) {
                         super.onIceCandidate(p0)
-                        peerConnection?.addIceCandidate(p0)
+                        Timber.d("Event-IceCandidate Generated")
+                        if (call.getCallState().value != CallState.ACTIVE){
+                            peerConnection?.addIceCandidate(p0)
```

Comment:
we should check if peerConnection exist before printing the log that the ICE candidate was added